src/main/java/frc/robot/commands/ShooterCommand.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class ShooterCommand extends CommandBase {
+  ShooterSubsystem shooterSubsystem;
+  private static final double SHOOTER_SPEED = 0.5; //TODO Find shooter speed
```

Comment:
Nit: The other files have constants like this at the top of the class declaration, before class members.

Reply:
Done.